app/Resources/views/common/department_dropdown.html.twig

```diff
@@ -1,7 +1,7 @@
 {% set dropdownId = dropdownId is defined ? dropdownId : 'departmentDropdown' %}
 <li>
     <a class="button tiny" data-dropdown="{{ dropdownId }}" aria-controls="{{ dropdownId }}"
-       aria-expanded="false">{{ currentDepartment is defined ? currentDepartment : 'Avdeling' }} <i
+       aria-expanded="false"><i class="fa fa-institution"></i> {{ currentDepartment is defined ? currentDepartment : 'Avdeling' }} <i
```

Comment:
Dette er kanskje litt pirkete, men hadde det sett bedre ut med `margin-right: 6px;` på ikonet i stedet for å ha et space mellom ikon og avdeling? (Det samme gjelder for semesteret)